lab-kayla/test/test.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
```

Comment:
**TODO FOR LOST POINTS**
Good start to your test module. Remember the instructions said you need three tests PER method, so you'll need 27 tests.